bot/cogs/cyber.py

```diff
@@ -368,6 +370,17 @@ def suffix_from_number(num):
             text += "We’re currently allocating students to their preferred locations so it’s an ongoing process!"
             text += " We’ll send out details of your location as soon as we can. It shouldn’t be too long!"
             await message.channel.send(text)
+        # Level Function Regex.
+        match = self.game_level_regex.match(message.content)
+        if match:
+            level_num, challenge_num = int(match.group(2)), int(match.group(3))
+            if match and message.channel.id == '532654287386443777':
```

Comment:
Here, you do `if match and ...`, but you're already inside an `if` block which guarantees that `match` is defined, so it should be safe to remove the checks here.

Reply:
omds i should check all of these b4 committing. ty will do this rn!